js/piwik.js

```diff
@@ -1138,15 +1138,7 @@ if (typeof Piwik !== 'object') {
             }
 
             function _elementInDocument(element) {
-                element = element.parentNode;
-
-                while (element) {
-                    if (element === documentAlias) {
-                        return true;
-                    }
-                    element = element.parentNode;
-                }
-                return false;
+                return documentAlias.documentElement.contains(element);
```

Comment:
I meant re Firefox 9+. 2012 there were still > 2% using it, I think 2015 it's still > 1.5% according to http://www.w3schools.com/browsers/browsers_firefox.asp . I personally do not see a problem there but just wanna be sure. Maybe we should mention somewhere which versions we try to support as we often had this topic